src/Microsoft.Framework.Runtime/Loader/AssemblyLoadContextFactory.cs

```diff
@@ -8,19 +8,17 @@ namespace Microsoft.Framework.Runtime.Loader
     public class AssemblyLoadContextFactory : IAssemblyLoadContextFactory
     {
         private readonly IAssemblyLoadContext _defaultContext;
-        private readonly IServiceProvider _serviceProvider;
 
         public AssemblyLoadContextFactory(IServiceProvider serviceProvider)
         {
             var accessor = serviceProvider.GetService(typeof(IAssemblyLoadContextAccessor)) as IAssemblyLoadContextAccessor;
-            _serviceProvider = serviceProvider;
             _defaultContext = accessor.Default;
         }
-
-        public IAssemblyLoadContext Create()
+        
```

Comment:
Use this: https://visualstudiogallery.msdn.microsoft.com/bcb3ef8e-a3a4-401e-9e63-fdf00b45268f